index.js

```diff
@@ -0,0 +1,139 @@
+document.addEventListener("DOMContentLoaded",()=>{
```

Comment:
Overall good implementation of HTML classes and methods to make workable Hangman game. I like the way that you divided your page with side divs and main div. I am looking forward to see what you will make for next projects/assignments. Keep it up!